### ISSUE TITLE
Investigate support for pytorch

### ISSUE THREAD

**Issue by raghakot** [raghakot/keras-vis#51]
All we need is iteration over layers, ability to take gradients with respect to some input in a graph, and abstractions for common keras backend calls.

It might be possible to make the lib support pytorch once these abstractions are in place.
